### PR TITLE
CORTX-6081 - Port m0spiel utilities and ST to python3

### PR DESCRIPTION
**Pull request by JugalPatil**
### Problem Statement
Python2 is EOL, port m0spiel to python3

### Testing
Unit and System Tests are working
Testing was performed with RPM

JIRA - https://jts.seagate.com/browse/CORTX-6081

### CODE REVIEW

File: utils/functions [hunk 1]
```diff
@@ -155,7 +155,7 @@ m0_default_xprt()
     echo $trans
 }
 
-## To get local nid address
+## Te get local nid address
```
Comment (huanghua78) [inline utils/functions:158]: Why is this needed?

Comment (JugalPatil) [reply utils/functions]: Thanks Hua, It was a typo not an intended change. I have reverted it back.

Comment (huanghua78) [reply utils/functions]: OK.

**Comment by mehjoshi** [comment]
@atulsdeshmukh2312 , @sanjognaik  Can you please review this change?

**Review by sanjognaik — APPROVED** [review]
Looks good.

**Review by atulsdeshmukh2312 — APPROVED** [review]
